portfolio/src/main/webapp/map.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Map</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
+    <script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBJb1ocfE_W74ua4gskytNLD4v0Shgrxa8&callback=initMap"></script>
+  </head>
+  <body onload="initMap();">
+    <div id="barTop">
+      <nav class="navbar navbar-expand-lg navbar-dark" style="background-color: #004047">
+        <a class="navbar-brand" href="index.html">Andrés's Portfolio</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" 
+            aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+          <div class="navbar-nav">
+          <a class="nav-item nav-link" href="index.html">Home</a>
+          <a class="nav-item nav-link" href="hobbies.html">Hobbies</a>
+          <a class="nav-item nav-link" href="comments.html">Comments</a>
+          <a class="nav-item nav-link active" href="#">Map<span class="sr-only">(current)</span></a>
+          </div>
+        </div>
+      </nav>
+    </div>
+    <div id="content">
+      <div id="title">
+        <h1>My Map</h1>
+        <h6>This is a map of my hometown: Monterrey!</h6>
+        <h6>It's a beautiful and modern city with lots of mountains sorrounding it!</h6>
```

Comment:
nit-spelling: surrounding

```suggestion
        <h6>It's a beautiful and modern city with lots of mountains surrounding it!</h6>
```